docs/program/apis/cloud.md

```diff
@@ -188,9 +188,293 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
+If nothing is passed, the location does not move.
+If an empty string is passed, the location moves up to the top level.
+
+**Raises:**
+
+- `GRPCError`: This error is raised if an invalid location ID is passed, or if one isn't passed and no location ID was provided at `AppClient` instantiation.
+
+**Returns:**
+
+- [(viam.proto.app.Location)](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Location): The newly updated location.
+
+```python {class="line-numbers linkable-line-numbers"}
+# The following line takes the location with ID "abc12abcde" and moves it
+# to be a sub-location of the location with ID "xyz34xxxxx"
+my_updated_location = await cloud.update_location(
+    location_id="abc12abcde",
+    name="",
+    parent_location_id="xyz34xxxxx"
+)
+
+# The following line changes the name of the location without changing its parent location
+my_updated_location = await cloud.update_location(
+    location_id="abc12abcde",
+    name="Land Before Robots"
+)
+
+# The following line moves the location back up to be a top level location without changing its name
+my_updated_location = await cloud.update_location(
+    location_id="abc12abcde",
+    name="",
+    parent_location_id=""
+)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.update_location).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### DeleteLocation
+
+Delete a {{< glossary_tooltip term_id="location" text="location" >}}.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `location_id` [(string)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): ID of the location to delete.
+
+**Raises:**
+
+- `GRPCError`: This error is raised if an invalid location ID is passed.
+
+**Returns:**
+
+- None
+
+```python {class="line-numbers linkable-line-numbers"}
+# Replace string with a valid location ID
+await cloud.delete_location(location_id="abc12abcde")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.delete_location).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ListLocations
+
+Get a list of all {{< glossary_tooltip term_id="location" text="locations" >}} under the organization you are currently authenticated to.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- (List[[viam.proto.app.Location](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.Location)]): The list of locations.
+
+```python {class="line-numbers linkable-line-numbers"}
+locations = await cloud.list_locations()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.list_locations).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### LocationAuth
+
+Get a location’s `LocationAuth` (location secret or secrets).
```

Comment:
```suggestion
Get a location’s location secret by ID.
```
Sorry, I know I approved already, but I think specifying by ID in description is good and would remove parenthesis. Optional but strongly suggest

Reply:
Hmm the LocationAuth isn't just the location secret; it also contains location ID and secret IDs. How about this
```suggestion
Get a location’s authentication information by location ID.
```